### PR TITLE
fix: rename staked amount to withdrawable stake

### PR DESCRIPTION
**Pull request by martinconic**
### Checklist

- [ ] I have read the [coding guide](https://github.com/ethersphere/bee/blob/master/CODING.md).
- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [ ] I have filled out the description and linked the related issues.

### Description
Renamed `stakedAmount` to `withdrawableStake`

### CODE REVIEW

File: pkg/api/staking_test.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func TestDepositStake(t *testing.T) {
 	t.Parallel()
 
 	txHash := common.HexToHash("0x1234")
-	minStake := big.NewInt(100000000000000000).String()
```
Comment (janos) [inline pkg/api/staking_test.go:29]: I am just curious, why is minStake lowered?

File: openapi/SwarmCommon.yaml [hunk 2]
```diff
@@ -634,7 +634,7 @@ components:
     GetStakeResponse:
       type: object
       properties:
-        stakedAmount:
```
Comment (janos) [inline openapi/SwarmCommon.yaml:637]: API spec version should be updated to reflect the breaking change.

**Review by janos — COMMENTED** [review]
beekeeper bee client needs to be updated for this api change.

File: pkg/api/staking_test.go (hunk 1 above)
Comment (martinconic) [reply pkg/api/staking_test.go]: Very good catch. My mistake, minStake did not get lowered. Fixed it

**Comment by martinconic** [comment]
> beekeeper bee client needs to be updated for this api change.

Already updated, please review

File: openapi/SwarmCommon.yaml (hunk 2 above)
Comment (martinconic) [reply openapi/SwarmCommon.yaml:637]: The API version was already updated for Bee 2.2. As I know it should be updated(only if there are any API changes) once per new Bee release.

**Comment by janos** [comment]
> > beekeeper bee client needs to be updated for this api change.
> 
> Already updated, please review

Beekeeper still fails on GetStakedAmount https://github.com/ethersphere/bee/actions/runs/10622306294/job/29446279002?pr=4796#step:19:31.

**Comment by martinconic** [comment]
> > > beekeeper bee client needs to be updated for this api change.
> > 
> > 
> > Already updated, please review
> 
> Beekeeper still fails on GetStakedAmount https://github.com/ethersphere/bee/actions/runs/10622306294/job/29446279002?pr=4796#step:19:31.

https://github.com/ethersphere/beekeeper/pull/413

**Review by janos — APPROVED** [review]

File: .github/workflows/beekeeper.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ env:
   SETUP_CONTRACT_IMAGE: "ethersphere/bee-localchain"
   SETUP_CONTRACT_IMAGE_TAG: "0.9.1-rc3"
   BEELOCAL_BRANCH: "main"
-  BEEKEEPER_BRANCH: "master"
+  BEEKEEPER_BRANCH: "fix/staked-amount-rename"
```
Comment (acha-bill) [inline .github/workflows/beekeeper.yml:17]: since the beekeeper PR has been merged, you can do a new beekeeper release and put this back to master

Comment (martinconic) [reply .github/workflows/beekeeper.yml:17]: It is in the works, just to have a new tag release on beekper

**Review by istae — APPROVED** [review]

**Pull request merged**